projects/angular/src/layout/_card.clarity.scss

```diff
@@ -169,27 +188,72 @@
     }
   }
 
-  .card-header,
-  .card-block,
-  .card-footer {
-    padding: card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal;
+  .card-block {
+    @include mixins.css-var(
+      padding,
+      clr-card-block-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+      variables.$clr-use-custom-properties
+    );
   }
 
-  .card-header,
-  .card-title {
+  .card-footer {
     @include mixins.css-var(
-      color,
-      clr-card-title-color,
-      card-variables.$clr-card-title-color,
+      padding,
+      clr-card-footer-padding,
+      card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
       variables.$clr-use-custom-properties
     );
+  }
+
+  h3 {
+    &.card-header {
+      @include mixins.css-var(
+        padding,
+        clr-card-header-padding,
+        card-variables.$card-children-padding-vertical card-variables.$card-children-padding-horizontal,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        color,
+        clr-card-header-title-color,
+        card-variables.$clr-card-title-color,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        font-size,
+        clr-card-header-title-font-size,
+        variables.$clr-h3-font-size,
+        variables.$clr-use-custom-properties
+      );
+    }
+  }
+
+  h4 {
+    &.card-title {
+      @include mixins.css-var(
+        color,
+        clr-card-content-title-color,
+        card-variables.$clr-card-title-color,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        font-size,
+        clr-card-content-title-font-size,
+        variables.$clr-h4-font-size,
+        variables.$clr-use-custom-properties
+      );
+    }
+  }
+
+  .card-header,
+  .card-title {
     @include mixins.css-var(
       font-weight,
       clr-card-title-font-weight,
       card-variables.$clr-card-title-font-weight,
       variables.$clr-use-custom-properties
     );
-    font-size: card-variables.$clr-card-title-font-size;
```

Comment:
Should be a default `font-size` in case the `.card-title` is not an `h3` or `h4`? The code examples on the website all have the `.card-title` as an `h3` or `h4`, but there are some non-examples that use a `div` instead. I don't know if the card title should alway be a heading element or not, but this seems like a breaking change when the shim is not applied that we should not ship in a non-major release.

Reply:
Yes, you are right. We don't want to block them from using different elements.